<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-N478O9TO" failures="1" tests="6" name="placeOrder.placeOrder" time="269.869" errors="0" timestamp="2023-06-17T21:44:21 IST" skipped="0">
  <testcase classname="placeOrder.placeOrder" name="launchApp" time="2.390"/>
  <system-out/>
  <testcase classname="placeOrder.placeOrder" name="checkPageLoad" time="0.168"/>
  <system-out/>
  <testcase classname="placeOrder.placeOrder" name="perform_Search" time="3.105"/>
  <system-out/>
  <testcase classname="placeOrder.placeOrder" name="perform_ProductCheckout" time="6.295"/>
  <system-out/>
  <testcase classname="placeOrder.placeOrder" name="perform_Login" time="20.855"/>
  <system-out/>
  <testcase classname="placeOrder.placeOrder" name="perform_payment" time="237.056">
    <failure message="Payment failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Payment failed
at org.testng.Assert.fail(Assert.java:99)
at placeOrder.placeOrder.perform_payment(placeOrder.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- perform_payment -->
  <system-out/>
</testsuite> <!-- placeOrder.placeOrder -->
